website/docs/docs/deploy/deployment-overview.md

```diff
@@ -52,11 +52,33 @@ For more details on both of these methods, including example implementations, ch
 
 ## Prefect
 
-If your organization is using [Prefect](https://www.prefect.io), use the [DbtShellTask](https://docs.prefect.io/api/latest/tasks/dbt.html#dbtshelltask) to schedule, execute and monitor your dbt runs.
+If your organization is using [Prefect]([url](https://www.prefect.io/)), the way you will run your jobs depends on the version you are on, and whether you are orchestrating dbt Cloud or core jobs. A variety of options are described below.
 
-Alternatively, you can use the supported [ShellTask](https://docs.prefect.io/api/latest/tasks/shell.html#shelltask) to execute dbt commands through the shell.
+<Lightbox src="https://user-images.githubusercontent.com/12350579/186269036-01f1090f-1887-433d-8331-ee7c8b315dd5.png" title="Prefect DAG using a dbt Cloud job run flow"/> 
+
+### On Prefect 2
+
+#### dbt Cloud
+
+Use the [trigger_dbt_cloud_job_run_and_wait_for_completion]([url](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion)) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [prefect UI]([url](https://docs.prefect.io/ui/overview/)).
```

Comment:
hey @belasobral93 fixing the url as the deploy link was directing this to a 'page not found'. below is the URL guidance:

❌ [prefect UI]([url](https://docs.prefect.io/ui/overview/)).
✅ [prefect UI](https://docs.prefect.io/ui/overview/).

```suggestion
Use the [trigger_dbt_cloud_job_run_and_wait_for_completion](https://prefecthq.github.io/prefect-dbt/cloud/jobs/#prefect_dbt.cloud.jobs.trigger_dbt_cloud_job_run_and_wait_for_completion) flow. As jobs are executing, you can poll dbt to see whether or not the job completes without failures, through the [Prefect UI](https://docs.prefect.io/ui/overview/).
```